src/main/kotlin/com/manimdsl/runtime/datastructures/stack/StackExecutor.kt

```diff
@@ -36,7 +36,8 @@ class StackExecutor(
     override val animationSpeeds: ArrayDeque<Double>,
     override val dataStructureBoundaries: MutableMap<String, BoundaryShape>,
     override val variableNameGenerator: VariableNameGenerator,
-    override val codeTextVariable: String
+    override val codeTextVariable: String,
+    private val locallyCreatedDynamicVariables: MutableSet<String>,
```

Comment:
if this is the same as the ones in BinaryTreeExecutor and ArrayExecutor, can we make it more consistent?